.github/workflows/deploy.yml

```diff
@@ -0,0 +1,36 @@
+name: Deploy modules
+on:
+  push:
+    branches:
+      - ghactions
+jobs:
+  build-modules:
+    runs-on: ubuntu-latest
+    environment: trial-572dca111144a5196a6b1
+    env:
+        CDF_CLUSTER: ${{ secrets.CDF_CLUSTER }}
+        CDF_PROJECT: ${{ secrets.CDF_PROJECT }}
+        IDP_CLIENT_ID: ${{ secrets.IDP_CLIENT_ID }}
+        IDP_CLIENT_SECRET: ${{ secrets.IDP_CLIENT_SECRET }}
+        IDP_TOKEN_URL: ${{ secrets.IDP_TOKEN_URL }}
+        IDP_SCOPES: ${{ secrets.IDP_SCOPES }}
+        IDP_AUDIENCE: ${{ secrets.IDP_AUDIENCE }}
+        CDF_URL: ${{ secrets.CDF_URL }}
+    name: Loadmaster
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+      - uses: snok/install-poetry@v1
+        with:
+          version: 1.6.1
+          virtualenvs-create: false
+      - name: Install dependencies
+        run: poetry install
+      - name: "Build the package"
+        run: |
+          poetry run python build.py --package
```

Comment:
What is the --package meant to be? It is not implemented from what I can see?
Could we do --run-tests here, so that the gh-action runs through build and deploy stages using a mock CDF client and thus fails quickly? If build.sh and deploy.sh then just uses pytest to execute their respective tests, it will be easy to verify locally that all tests pass (and by using pytest, we can run individual tests from VSC).

Reply:
the --package was a dead end, will be removed

Reply:
Regarding testing: will pick this up in a separate PR #32